package/post-install.sh

```diff
@@ -11,17 +11,25 @@ CUSTOM_INI_FILE_NAME='ddtrace-custom.ini'
 
 PATH="${PATH}:/usr/local/bin"
 
-if [[ -z "$DD_TRACE_PHP_BIN" ]]; then
-    DD_TRACE_PHP_BIN=$(command -v php)
```

Comment:
Wasn't able to understand which sort of problem `command -v` was causing on alpine. Moving to `$(which php || true)` but taking note of it to look into it asap